pyClient/zeth/joinsplit.py

```diff
@@ -326,7 +327,7 @@ def get_dummy_input_and_address(
         value=ZERO_UNITS_HEX,
         rho=get_dummy_rho(),
         trap_r=trap_r_randomness())
-    dummy_note_address = 7
+    dummy_note_address = 2 ** constants.ZETH_MERKLE_TREE_DEPTH - 1
```

Comment:
Note that the dummy note address could be chosen arbitrarily, because if the note value is 0, then the recomputed merkle root in the tree is not compared to the actual merkle root (i.e. the merkle auth path is not checked). May be worth adding a comment to clarify that this address is picked arbitrarily

Reply:
Added a comment, and switched to 0.

Reply:
Thanks